pysat/tests/classes/cls_instrument_library.py

```diff
@@ -616,8 +624,11 @@ def test_clean_warn(self, clean_level, inst_dict, caplog):
 
         return
 
+    # TODO(v3.3.0): remove mark.new_tests when ecosystem is fully compliant
+    # for this test
```

Comment:
```suggestion
    # TODO(#XXX): remove mark.new_tests at v3.3.0
```
Why don't you create an issue for this and tag it to the v3.3.0 milestone?